editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -31,10 +31,26 @@
 #include "sprite_frames_editor_plugin.h"
 
 #include "core/io/resource_loader.h"
+#include "core/os/keyboard.h"
 #include "core/project_settings.h"
 #include "editor/editor_settings.h"
 #include "scene/3d/sprite_3d.h"
 
+void SpriteFramesEditor::_process_input(const Ref<InputEvent> &p_event, Node *p_from) {
+
+	Ref<InputEventKey> k = p_event;
+
+	if (k.is_valid() && k->is_pressed() && k->get_scancode() == KEY_DELETE && !k->is_echo()) {
```

Comment:
Can this be replaced with a shortcut?
Maybe even something like: https://github.com/godotengine/godot/blob/1f560c4679e1434fbc6fc442c3ad209ffd8b592e/editor/plugins/tile_set_editor_plugin.cpp#L425